labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Chirpus Inc.</title>
+    <link href="mobilefirststyle.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+
+<body>
+
+    <h3 class="logo">chirpus inc.</h3>
+    <nav>
+        <div class="menu">
+            <ul>
```

Comment:
If you use a`<ul>`, you should only use `<li>`'s inside it.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul